Emby.Server.Implementations/LiveTv/TunerHosts/HdHomerun/HdHomerunHost.cs

```diff
@@ -661,18 +661,18 @@ public async Task<List<TunerHostInfo>> DiscoverDevices(int discoveryDurationMs,
                 // Need a way to set the Receive timeout on the socket otherwise this might never timeout?
                 try
                 {
-                    await udpClient.SendToAsync(discBytes, 0, discBytes.Length, new IPEndPoint(IPAddress.Parse("255.255.255.255"), 65001), cancellationToken).ConfigureAwait(false);
+                    await udpClient.SendToAsync(discBytes, new IPEndPoint(IPAddress.Parse("255.255.255.255"), 65001), cancellationToken).ConfigureAwait(false);
```

Comment:
How about `IPAddress.Broadcast` instead of `IPAddress.Parse("255.255.255.255")`?